tools/ci_build/github/azure-pipelines/templates/py-packaging-selectable-stage.yml

```diff
@@ -152,10 +152,16 @@ stages:
           filename: 'C:\Program Files\Intel\openvino_2021.4.752\bin\setupvars.bat'
           modifyEnvironment: true
 
-      - script: |
-          python -m pip install -q setuptools wheel numpy==1.21.6
-        workingDirectory: '$(Build.BinariesDirectory)'
-        displayName: 'Install python modules'
+      - task: PythonScript@0
+        inputs:
+          scriptSource: inline
+          script: |
+            import sys
+            version = 'numpy==1.21.6' if sys.version_info < (3, 11) else 'numpy==1.24.2'
```

Comment:
I don't understand this part. Here you only check if numpy is installed or not and if the version is expected. It doesn't actually install numpy. Then when numpy was installed?

Reply:
This part is to check if python version is less than 3.11. if it is less than 3.11 it will install numpy 1.21.6 else it will install 1.24.2 using pip on line 162